spec.bs

```diff
@@ -221,6 +226,55 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
     [=interest group ad/render url=].
   1. The [=string/length=] of |ad|'s [=interest group ad/metadata=] if the field is not null.
 
+<div algorithm>
+
+To <dfn>check interest group permissions</dfn> given an [=url/origin=]
+|ownerOrigin|, an [=url/origin=] |frameOrigin|, and a [=boolean=] |isJoin|:
+1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.
+1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]
+  with |ownerOrigin| and |frameOrigin|.
+1. Let |resource| be the result of [=fetching resource=] with |permissionsUrl|

```

Comment:
remove this step (if we do want to create its own request)?

Reply:
I'm not sure what you mean -- if we don't fetch the resource, this amounts to nothing, right?

Reply:
oh I think you have the fetching below (at line 241), right? And at line 240, you set |resource| to null, which overwrites this |resource|? This line seems like some old thing that forgets to be removed, when lines 238+ are added.

Reply:
Oh, I see, this was just a leftover. Deleted.